src/containers/FiveDay.js

```diff
@@ -1,20 +1,19 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router';
+// import { Link } from 'react-router';
 import currentWeather from '../data/fake-current';
 
 export default class FiveDay extends Component {
```

Comment:
Does this component have any state? If not, than why not use a stateless component?